sdk/src/client/api/types.rs

```diff
@@ -160,7 +160,7 @@ impl TryFrom<RemainderDataDto> for RemainderData {
 
     fn try_from(dto: RemainderDataDto) -> Result<Self, Self::Error> {
         Ok(Self {
-            output: Output::try_from(dto.output)?,
```

Comment:
Can be a from now ?

Reply:
Yes, but I would do other types in an own PR so this here doesn't get bigger. Already opened an issue https://github.com/iotaledger/iota-sdk/issues/1713

Reply:
I mean we can remove the whole dto here

Reply:
Ok sir